hive/metadata.csv

```diff
@@ -10,18 +10,6 @@ hive.metastore.partition.created,rate,10,item,,Total number of created partition
 hive.metastore.partition.deleted,rate,10,item,,Total number of deleted partition.,0,hive,hms partition deleted,
 hive.metastore.partition.init,rate,10,item,,Number of initialized partition.,0,hive,hms partition init,
 hive.metastore.directsql_errors,gauge,10,unit,,Number of SQL error.,0,hive,hms sql error,
-hive.metastore.memory.heap.init,gauge,10,byte,,Memory used at the initialization by the metastore.,0,hive,hms mem heap init,
```

Comment:
Should we update the `hive.server.memory.*` metric descriptions to mention that the `metastore` memory metrics are also included under those metrics?

Reply:
I added some language around the metric description. Let me know if it sounds confusing or not.